PTGLdynamics/codes/plotting-scripts/PyMolHeatmapVisualisation.py

```diff
@@ -42,6 +43,11 @@ def log(message, level=""):
             global output_file
             output_file.write(message + "\n")
 
+def new_directory(name):
```

Comment:
I would suggest renaming the method to `create_directory` or something similar that tells you clearer what the method does.